api.ts

```diff
@@ -966,11 +966,19 @@ export interface SyncFormulaResult<K extends string, L extends string, SchemaT e
 
 /**
  * Type definition for the parameter used to pass in a batch of updates to a sync table update function.
- * @hidden
  */
 export interface SyncUpdate<K extends string, L extends string, SchemaT extends ObjectSchemaDefinition<K, L>> {
+  /**
+   * The previous value of the row.
+   */
   previousValue: ObjectSchemaDefinitionType<K, L, SchemaT>;
+  /**
+   * The new value of the row, with the user edits applied.
+   */
   newValue: ObjectSchemaDefinitionType<K, L, SchemaT>;
+  /**
+   * The fields of the object that have been updated.
```

Comment:
Nit: we use `object` here but `row` above.  Should these be consistent?

Reply:
Good catch, done.